Makefile

```diff
@@ -35,6 +35,7 @@ TESTFLAGS    :=
 STRESSFLAGS  := -stderr -maxfails 1
```

Comment:
in the commit message:
> make stress should in the package directory

missed the word run
> When using `make test` the tests runs in the package directory. When using `make

mixing singular and plural in "tests runs"
> stress`, it ran in the current (cockroach) directory. If we are passing a path

"it" is singular
> via TESTFLAGS, the distinction is important. This change makes the stress
program run under the package directory.

"stress program" singular again